site/docs/get-started/docker.md

```diff
@@ -8,18 +8,20 @@ ManageIQ project makes available on the [Docker Hub](https://hub.docker.com/r/ma
 This is a great option if you have a Linux PC (but it works everywhere Docker
 is available).
 
+{% assign release = site.data.releases["stable"] %}
+
 ### Step 1: Download and deploy the appliance
 
 Pull the ManageIQ docker image:
 
 ```bash
-$ sudo docker pull manageiq/manageiq:latest-darga
```

Comment:
The "latest" tag will be valid, but I think it's better to use a stable image for the getting started guide. If the user is more experienced they can find out easily the other options to use, including latest.